src/components/attendee/Button/Button.module.scss

```diff
@@ -14,6 +14,11 @@
   white-space: nowrap;
   cursor: pointer;
   border: none;
+
+  &:focus {
+    // removes blue highlighting on mobile tap
+    -webkit-tap-highlight-color: transparent;
```

Comment:
[Q] What happens on Firefox which doesn't have a Webkit engine (on Android at least)?

Reply:
Firefox doesn't seem to have this as a feature, so no blue highlight appears there with this thing added/removed